addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -2850,692 +2850,6 @@ return Timers;
-        // one are both comments wrote by the same author at the same minute
-        // and in the same document (users can now post message in documents
-        // directly from a channel that follows it)
-        var prevMessage;
-        _.each(messages, function (message) {
-            if (
-                // is first message of thread
-                !prevMessage ||
-                // more than 1 min. elasped
-                (Math.abs(message.getDate().diff(prevMessage.getDate())) > 60000) ||
-                prevMessage.getType() !== 'comment' ||
-                message.getType() !== 'comment' ||
-                // from a different author
-                (prevMessage.getAuthorID() !== message.getAuthorID()) ||
-                (
-                    // messages are linked to a document thread
-                    (
-                        prevMessage.isLinkedToDocumentThread() &&
-                        message.isLinkedToDocumentThread()
-                    ) &&
-                    (
-                        // are from different documents
-                        prevMessage.getDocumentModel() !== message.getDocumentModel() ||
-                        prevMessage.getDocumentID() !== message.getDocumentID()
-                    )
-                )
-            ) {
-                displayAuthorMessages[message.getID()] = true;
-            } else {
-                displayAuthorMessages[message.getID()] = !options.squashCloseMessages;
-            }
-            prevMessage = message;
-        });
-
-        if (modeOptions.displayOrder === ORDER.DESC) {
-            messages.reverse();
-        }
-
-        this.$el.html(QWeb.render('im_livechat.legacy.mail.widget.Thread', {
-            thread: thread,
-            displayAuthorMessages: displayAuthorMessages,
-            options: options,
-            ORDER: ORDER,
-            dateFormat: time.getLangDatetimeFormat(),
-        }));
-
-        _.each(messages, function (message) {
-            var $message = self.$('.o_thread_message[data-message-id="' + message.getID() + '"]');
-            $message.find('.o_mail_timestamp').data('date', message.getDate());
-
-            self._insertReadMore($message);
-        });
-
-        if (shouldScrollToBottomAfterRendering) {
-            this.scrollToBottom();
-        }
-
-        if (!this._updateTimestampsInterval) {
-            this.updateTimestampsInterval = setInterval(function () {
-                self._updateTimestamps();
-            }, 1000 * 60);
-        }
-
-        this._renderMessageNotificationPopover(messages);
-        if (thread.hasSeenFeature()) {
-            this._renderMessageSeenPopover(thread, messages);
-        }
-    },
-
-    /**
-     * Render thread widget when loading, i.e. when messaging is not yet ready.
-     * @see /mail/init_messaging
-     */
-    renderLoading: function () {
-        this.$el.html(QWeb.render('im_livechat.legacy.mail.widget.ThreadLoading'));
```

Comment:
ci/security due to this move

Reply:
This one is fine, especially the value is coming from a template.